cli/src/api/pcluster_api.py

```diff
@@ -43,6 +44,21 @@ def __repr__(self):
         return json.dumps(self.__dict__)
 
 
+class ImageInfo:
+    """Minimal representation of a building image."""
+
+    def __init__(self, imagebuilder_manager: ImageBuilderManager):
```

Comment:
If all the information are from the stack you can pass the imagebuilder_manager.stack rather than passing the entire manager. This helps for mocking and for testing.

Reply:
yes, after thinking a while, here I still pass the imagebuilder_manager object, because I decide to move imagebuild_status to manager rather than stack, imagebuild_status should be imagebuilder_manager's attribute.